services/director-v2/src/simcore_service_director_v2/modules/db/repositories/comp_tasks.py

```diff
@@ -27,34 +23,23 @@
 
 logger = logging.getLogger(__name__)
 
-
-@log_decorator(logger=logger)
-def _get_fake_service_details(
-    service: ServiceKeyVersion,
-) -> Optional[ServiceDockerData]:
-
-    if "file-picker" in service.key:
-        file_picker_outputs = {
-            "outFile": {
-                "label": "File",
-                "displayOrder": 0,
-                "description": "Chosen File",
-                "type": "data:*/*",
-            }
-        }
-        return ServiceDockerData(
-            **service.dict(),
-            name="File Picker",
-            description="File Picker",
-            authors=[
-                Author(name="Odei Maiz", email="maiz@itis.swiss", affiliation="IT'IS")
-            ],
-            contact="maiz@itis.swiss",
-            inputs={},
-            outputs=file_picker_outputs,
-            type=ServiceType.FRONTEND,
-        )
-    return None
+#
+# This is a catalog of front-end services that are translated as tasks
+#
+# The evaluation of this task is already done in the front-end
+# The front-end sets the outputs in the node payload and therefore
+# no evaluation is expected in the backend.
+#
+# Examples are nodes like file-picker or parameter/*
+#
+_FRONTEND_SERVICES_CATALOG: Dict[str, ServiceDockerData] = {
+    meta.key: meta
+    for meta in iter_service_docker_data()
+    if any(name in meta.key for name in ["file-picker", "parameter"])
+}
+assert (  # nosec
+    len(_FRONTEND_SERVICES_CATALOG) == 4
+), "If this fails, review filter above and update"  # nosec
```

Comment:
This is a bit weird. Would only trigger if the service names match "file-picker" or "parameter". Or is that the intention? And what about the node grouping service?

Reply:
If somebody adds more front-end services in the catalog, probably the filter I use (which is not very solid) will cause troubles.

this assert intends to warn developer to review that filter again and update it to the new situation or even refactor it.